docker-compose/_posts/2017-07-05-building-a-compose-file-for-runnable.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Building a Compose File for Runnable
+short: Building a Compose File for Runnable
+category: docker-compose
+order: 4
+---
+
+Runnable can deploy full-stack environments for branches of any repository that contains a Dockerfile. Let’s go over some practices you should follow in your Compose files to make this work for your team.
```

Comment:
I think it might be more straight-forward to say, "We can accept most Docker Compose files, but some modifications may be necessary to run as expected on Runnable".